onboarding/sessions/tour-of-artsy.md

```diff
@@ -0,0 +1,162 @@
+      - Online-only ("Timed")
+      - Online + Live ("LAI" == "Live Auction Integration", sometimes just called "Live Auctions")
+      - Online + Event ("Benefit auction live event")
+    - Reference:
+      - [Auction (Sale) model](https://github.com/artsy/gravity/blob/master/app/models/domain/sale.rb)
+      - [Auctions app code](https://github.com/artsy/force/tree/master/src/desktop/apps/auctions)
+      - [Administered in Ohm](http://auctions-staging.artsy.net/)
+  - **Galleries**
+    - Organized by "Partner category"
+    - Reference:
+      - [Partner model](https://github.com/artsy/gravity/blob/master/app/models/domain/partner.rb)
+      - [Galleries app code](https://github.com/artsy/force/tree/master/src/desktop/apps/galleries_institutions)
+      - [Administered in Vibrations](https://admin-partners-staging.artsy.net/)
+  - **Fairs**
+    - Fair artworks are published a couple of weeks before the fair begins ("Fair preview")
+    - Reference:
+      - [Fair model](https://github.com/artsy/gravity/blob/master/app/models/domain/fair.rb)
+      - [Fairs app code](https://github.com/artsy/force/tree/master/src/desktop/apps/fairs)
+      - [Administered in Waves](https://admin-fairs-staging.artsy.net/)
+  - **Magazine**
+    - Curated by our Editorial team
+    - Reference:
+      - [Administered and modeled in Positron](https://github.com/artsy/positron)
+      - [Articles app code](https://github.com/artsy/force/tree/master/src/desktop/apps/articles)
+
+4. Browse individual entities
+  - **Artwork page**
+    - Goal of the artwork page is to give the collector information about the artwork, confidence in their buying decision
+    - Reference:
+      - [Artwork model](https://github.com/artsy/gravity/blob/master/app/models/domain/artwork.rb)
+      - [Artwork app code](https://github.com/artsy/force/tree/master/src/desktop/apps/artwork)
+  - **Artist page**
+    - Recently made this page responsive. Is a top entry-point for "shopping" for an artist's work
+    - Reference:
+      - [Artist model](https://github.com/artsy/gravity/blob/master/app/models/domain/artist.rb)
+      - [Artist app code (Force)](https://github.com/artsy/force/tree/master/src/desktop/apps/artist)
+      - [Artist app code (Reaction)](https://github.com/artsy/reaction/tree/master/src/Apps/Artist)
+  - **Auction page**
+    - Reference:
+      - [filter/sale_artworks API](https://github.com/artsy/gravity/blob/master/app/api/v1/filter_endpoint.rb)
+      - [Auction app code](https://github.com/artsy/force/tree/master/src/desktop/apps/auction)
+  - **Gallery profile page**
+    - Vanity URL since it may be shared
+    - Reference:
+      - [App code](https://github.com/artsy/force/tree/master/src/desktop/apps/partner)
+  - **Article page**
+    - Many different types of articles, determines formatting/content options
+    - Reference:
+      - [App code](https://github.com/artsy/force/tree/master/src/desktop/apps/article)
+  - **Fair microsite**
```

Comment:
Is https://github.com/artsy/vanity the repo that backs all of these vanity URLs? If so, thoughts on including this as a reference to here, and maybe the `*Gallery profile page**` page above?